docs/body.md

```diff
@@ -748,39 +615,336 @@ when this happens.
+
+*   The RabbitMQ-Ceilometer service
+
+    `RabbitMQ-Ceilometer` is typically only needed for legacy (pre 3.0,0 ZenPack) systems.
+
+    Version 2.4.0 of this zenpack introduces a new service, `RabbitMQ-Ceilometer`.
+    This is a dedicated instance of RabbitMQ on each collector which is used solely
+    for integration with Ceilometer, rather than using the standard `RabbitMQ`
+    service that is used by Zenoss itself. This better distributes any load as
+    well as providing better support for distributed collector scenarios where the
+    target OpenStack environment might not have network access to the central
+    Zenoss servers.
+
+*   Zenoss collectors need direct access to the NovaHost devices in order to
+    model them. In some setups like RHOSP's overcloud, where direct
+    access to the overcloud network is not accessible by default, this can be a
+    challenge. Discussion of how to achieve connectivity to overcloud is
+    out of scope.
+
+### Optional Steps
+
+#### VM State Changes
+
+By default, instance state changes will be captured by Zenoss when
+certain events occur, for example, when an instance is shut down, the
+state change to SHUTDOWN will be reflected in Zenoss.
+
+However, certain state changes that don't correspond to another defined
+event may not be picked up until the next time Zenoss models the
+environment.
+
+If you would like to reduce the likelihood of this occurring, you can
+configure OpenStack Nova to send an event (through Ceilometer) to Zenoss
+whenever any VM state change occurs by adding the following to
+/etc/nova/nova.conf on all Nova hosts:
+
+      [notifications]
+      notify_on_state_change=vm_and_task_state
+
+      ...
+
+      [oslo_messaging_notifications]
+      driver = messagingv2
+
+Save /etc/nova/nova.conf and restart nova services.
+
+Note that notify_on_state_change will cause increased event load,
+both on OpenStack and Zenoss, and additional processing within the event
+transforms in Zenoss to keep the model consistent. Since most instance
+changes will still be caught without this option enabled, it is
+recommended to leave notify_on_state_change disabled if your
+OpenStack environment is very large.
+
+#### VM State Changes RHOSP
+
+RHOSP sets these values correctly.
+
+
+#### Network Events
+
+The OpenStack Infrastructure ZP pulls neutron events for Networks and Routers. However,
+if Neutron is not configured properly to send those events, they cannot be retrieved.
+If you are missing events, checks your OpenStack environment's
+/etc/neutron/neutron.conf.
+
+It should have the following for traditional deployments:
+
+```
+[oslo_messaging_notifications]
+driver = messagingv2
+topics = notifications
+```
+
+For RHOSP 13+ the driver configuration are usually set correctly to
+`messagingv2`. If not you must ensure all templates correctly specify:
+
+    NotificationDriver: 'messagingv2'
+
+Details of how to prepare the templates are beyond the scope of this document.
```

Comment:
Again, let's just move the part above into the manual config section (and perhaps simplify it, just tell them they need to confirm that these settings are there), and assume that RHOSP users don't need to worry about it.  Which I think is true, unless they've done some weird customization.